website_sale_product_minimal_price/__manifest__.py

```diff
@@ -13,5 +13,19 @@
     "application": False,
     "installable": True,
     "depends": ["website_sale"],
-    "data": ["views/assets.xml", "views/templates.xml"],
+    "data": ["views/templates.xml"],
+    "assets": {
+        "web.assets_frontend": [
+            "/web/static/src/legacy/js/fields/field_utils.js",
+            "/website_sale_product_minimal_price/static/src/js"
+            "/website_sale_product_minimal_price.js",
+            "/website_sale_product_minimal_price/static/src/js"
+            "/website_sale_product_price_scale.js",
+        ],
+        "web.assets_tests": [
+            "/website_sale_product_minimal_price/static/src/js/tour.js",
+            "/website_sale_product_minimal_price/static/src/js"
+            "/test_product_with_no_prices_tour.js",
+        ],
+    },
```

Comment:
Why not all the assets have the complete path reference?

Reply:
Take a closer look :wink: 

(it could be solved putting the sources in different folders and wildcarding the js files inside, though)

Reply:
:sweat_smile: ups sorry